packages/react-native-lockdown/LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2023 Consensys Software
```

Comment:
```suggestion
Copyright (c) 2024 Consensys Software
```

Reply:
- the year should be the year you started the work, not the current year
- rename to `LICENSE` since it's not markdown

Reply:
thanks for
- https://github.com/LavaMoat/LavaMoat/pull/1700

renamed it in this PR too